Source/UnitEx.cpp

```diff
@@ -170,14 +183,36 @@ struct OP2Unit
 	short timerESG;
 	short unknown8;
 	int unknown9;
-	char unknown10;
+	char workersInTraining;
 	char bayItem[6];
 	char unknown11;
 	int unknown12;
 	short unknown13;
 	int objectOnPad;
-	short unknown14;
-	int unknown15;
+	int launchPadCargo; // Arklon says this should be an int32. unknown15 would be int16?

```

Comment:
If we are pretty sure this is correct, lets just pull the comment in the final merge since this pull request will solve.

Reply:
The problem here is the data is context dependent. The meaning (and type/size) of the fields depends on the type of unit. We would need to overlay several structs with a union to accurately describe all `Unit` fields.

I'd be fine with this change being merged, and the comment being stripped. Though either way it's going to be wrong in some context. We can address that in a future issue.